static/app/components/organizations/timeRangeSelector/timeRangeItemLabel.tsx

```diff
@@ -0,0 +1,11 @@
+import styled from '@emotion/styled';
+
+import space from 'sentry/styles/space';
+
+const TimeRangeItemLabel = styled('div')`
```

Comment:
Pulled this out from `selectorItems.tsx` to better import from `timeRangeAutoCompleteFilter.tsx`